composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.6",
         "apishka/cs": "~1.0.0",
-        "composer-plugin-api": "~1.0.0",
+        "composer-plugin-api": "^1.0.0",
```

Comment:
`~1.1.0` would be more constant with the other version constraints in here. The intention is to limit the minor version as far as I can see.